rust/src/lib.rs

```diff
@@ -542,6 +632,11 @@ pub unsafe extern "C" fn zcashlc_free_ffi_uuid(ptr: *mut FfiUuid) {
 /// Adds a new account to the wallet by importing the UFVK that will be used to detect incoming
 /// payments.
 ///
+/// Derivation metadata may optionally be included. To indicate that no derivation metadata is
+/// available, the `seed_fingerprint` argument should be set to the null pointer and
+/// `hd_account_index` should be set to the value `u32::MAX`. Derivation metadata will not be
+/// stored unless both the seed fingerprint and the HD account index are provided.
```

Comment:
Do we want to just not store it, or should we return an error if this occurs? It would likely indicate a programming error.

Reply:
An error sounds like a reasonable solution.